active/website.md

```diff
@@ -0,0 +1,62 @@
+# Website Working Group
+
+## Scope of responsibilities
+
+This working group will manage Django's official website, **djangoproject.com** on behalf of the Django Software Foundation
```

Comment:
You probably need to clarify "manage" a bit -- we already have an ops team (hi @django/ops-team) that does a few things website-related. Delineating the devision of labor here, at least roughly, would be a good idea.

Reply:
I need help to clarify this, I'm not sure what the @django/ops-team 's tasks are here.

Reply:
I would recommend something like:

```suggestion
This is a replacement of the current maintainer team @django/djangoproject-com-maintainters. The team will own maintenance of the website codebase, and liaise with the @django/ops-team for production infrastructure considerations.
```

Deployments in particular feel to me like something that ought to be automated, rather than depending on any one team.

Reply:
Thank you both for your input here.